kpi/templates/registration/activation_email.txt

```diff
@@ -1,13 +1,15 @@
 {% load i18n %}
-{% trans "Thanks for signing up with KoBoToolbox!" %}
+{% trans "Dear" %} {{ user }},
 
-{% trans "Confirming your account will give you access to KoBoToolbox applications. Please visit the following URL to finish activation of your new account." %}
+{% trans "Thank you for signing up with KoBoToolbox!" %}
 
-{{ kpi_protocol }}://{{ site.domain }}/accounts/activate/{{activation_key}}
+{% trans "Confirming your account will give you full access to all KoBoToolbox applications. Please visit the following URL to finish the activation of your new account:" %}
 
-{% trans "Your username is: " %} {{ user }} 
+{{ kpi_protocol }}://{{ site.domain }}/accounts/activate/{{activation_key}}/
 
-{% trans "Please visit https://community.kobotoolbox.org/ to find information on how to get started. There you can also post questions to the community (recommended) or to us directly." %}
+{% trans "Your username is:" %} {{ user }}
```

Comment:
I added this explicit "Your username is: blah" from Anji's #2622 because I thought it'd be worthwhile to hammer this information home, as opposed to having it only in the "Dear blah," salutation. @tinok, are you okay with the placement and wording?

Reply:
Sounds good, I had the same thought in retrospect. We could even make it 

```
{% trans "Your username is: " %} {{ user }} 
{% trans "Your KoBoToolbox server is: "%} {{ kpi_protocol }}://{{ site.domain }}
```